### ISSUE TITLE
[Bug] DashChat is incompatible to Flutter v2.0.0

### ISSUE THREAD

**Issue by LostInDarkMath** [fayeed/dash_chat#163]
**Describe the bug**
I upgraded to Flutter v2.0.0 and after that I get a version solving problem, since `flutter_localizations` depends on `intl` 1.7.0, which is currently not supported by dash_chat.

**To Reproduce**
1. Install Flutter 2.0.0
2. Create a project with at least this depencies:
   ```yaml
   dependencies:
     flutter:
       sdk: flutter
     flutter_localizations: 
       sdk: flutter

     intl: ^0.17.0 
     dash_chat: ^1.1.15  
   ```
2. Run `flutter pub get`
3. See error
```
Because dash_chat >=1.0.10 depends on intl ^0.16.0 and foo depends on intl ^0.17.0, dash_chat >=1.0.10 is forbidden.
So, because foo depends on dash_chat ^1.1.15, version solving failed.
Running "flutter pub get" in foo...                              
pub get failed (1; So, because foo depends on dash_chat ^1.1.15, version solving failed.)
```

If I set `intl` to 1.16.0, which is supported by dash_chat, I get this error:
```
Because foo depends on flutter_localizations any from sdk which depends on intl 0.17.0, intl 0.17.0 is required.
So, because foodepends on intl ^0.16.1, version solving failed. 
Running "flutter pub get" in foo...                              
pub get failed (1; So, because foo depends on intl ^0.16.1, version solving failed.)
```

**Expected behavior**
No version solving Problems with `flutter pub get`.

**Desktop (please complete the following information):**
 - OS: Ubuntu
 - Flutter Version: 2.0.0
 - Dart Version: 2.1.12
 - Dash Chat Version: 1.1.15

**Comment by diegogarciar** [comment]
@LostInDarkMath I created my own branch just for this. I just updated the dependencies, but we need to migrate this package to null safety. I can try but might take me a while, is there someone else more determine on migrating as this package to null safety? It might not be updated by the author as he is not working on the package full time anymore.


```
dash_chat:
    git:
      url: git://github.com/diegogarciar/dash_chat.git

```

**Comment by diegogarciar** [comment]
Never mind, I started, expect a pull request later today.

**Comment by diegogarciar** [comment]
I have submitted the pull request, you can use my branch in the meantime at https://github.com/diegogarciar/dash_chat

I literally spent on it the time diff between my comments. I tested the example on iOS and it worked, please test it on your implementation rigorously and let me know if there's a bug.

**Comment by fayeed** [comment]
@jonchui Package is updated to `1.1.15-nullsafety`, thanks to @diegogarciar.

**Comment by diegogarciar** [comment]
You're welcome anytime @fayeed, and if you need help in the future, please let me know.

**Comment by diegogarciar** [comment]
Btw, I think its better if you also release a `2.0.0` without `-nullsafety`

**Comment by fayeed** [comment]
Yeah, I am planning to release an alpha version soon for the v2.

**Comment by jinfagang** [comment]
@fayeed is there a timeline of releasing v2?

**Comment by diegogarciar** [comment]
I meant that by adding null safety, you should release a major version number change, I didn't mean your planned version for V2, I suggest should either call it v3 or just bump the new one to 1.2.0.

**Comment by fayeed** [comment]
@jinfagang You can expect an alpha version in 2 to 3 weeks.

**Comment by fayeed** [comment]
@diegogarciar makes sense I will release `1.2.0` in an hour or two.

**Comment by m-j-g** [comment]
@DennisAshford this repo is no longer being updated, I think most of us moved over to https://pub.dev/packages/flutter_chat_ui

**Comment by SebastienBtr** [comment]
> Was this ever released? I still see 1.1.16 as the latest version on pub.dev?

As you can see in the readme:
